### PR TITLE
Added custom inline-image plugin & modified build.js

### PR DESCRIPTION
**Pull request by anujpunjani**
This PR resolves issue #98

### CODE REVIEW

File: tasks/check-for-image-url.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+const PLUGIN_NAME = 'gulp-inline-images';
+const MIME_TYPE_REGEX = /.+\/([^\s]*)/;
+const INLINE_ATTR = 'inline';
+const NOT_INLINE_ATTR = `!${INLINE_ATTR}`;
+
+function inlineImg(options = {}) {
+  const selector = options.selector || 'img[src]';
+  const attribute = options.attribute || 'src';
+  const getHTTP = options.getHTTP || false;
```
Comment (danmindru) [inline tasks/check-for-image-url.js:21]: @anujpunjani ah cool - so I see this is a new feature then :) 
As I can read, if we would flip this to true there will be a check / inlining for remote urls, which is quite neat.
Sometimes users will want that, sometimes they wont (depending on the size of the image) 

Do you have any suggestion how to inline remote images in a user-friendly way?

I am thinking, we could allow some sort of hint in the url to bypass this option.

So in `conf.js`, the url could have a value of:
- `"url": "https://ex.com/img.png?inline"` to inline as base64
- `"url": "https://ex.com/img.png"` to use the plain url

What do you think?

**Review by danmindru — COMMENTED** [review]
hi @anujpunjani, sorry for the delayed review - has been a crazy couple of weeks!
I did some minor style changes, but at a glance it seems to work great - nice work!

There's just the minor comment below, otherwise should be good to merge.

File: tasks/check-for-image-url.js (hunk 1 above)
Comment (anujpunjani) [reply tasks/check-for-image-url.js:21]: If I understand correctly, in conf.json, we could provide `"inline": true` as a property and use it as a parameter on line 37 of build.js like `.pipe(inlineImg({ getHTTP: confItems[0]["inline"] }))`.

Comment (danmindru) [reply tasks/check-for-image-url.js:21]: Yeah, your idea is even better :) 
Would be awesome to get that in

Comment (anujpunjani) [reply tasks/check-for-image-url.js:21]: I have added some changes regarding this in the commit [afa48c0](https://github.com/danmindru/responsive-html-email-signature/pull/100/commits/afa48c094ee8d43f2cadb37e508a3470c8e3471a)

Comment (danmindru) [reply tasks/check-for-image-url.js:21]: Awesome! Works great :-) 
Thanks again for the contribution, merged! 🎉

Comment (anujpunjani) [reply tasks/check-for-image-url.js:21]: This was a new learning experience for me as well. I am glad I could contribute.😄

**Pull request merged**